plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/BaseTrinoCatalogTest.java

```diff
@@ -66,14 +73,22 @@
 import static io.trino.spi.StandardErrorCode.NOT_SUPPORTED;
 import static io.trino.spi.type.BigintType.BIGINT;
 import static io.trino.sql.planner.TestingPlannerContext.PLANNER_CONTEXT;
-import static io.trino.testing.TestingConnectorSession.SESSION;
 import static io.trino.testing.TestingNames.randomNameSuffix;
 import static java.util.Locale.ENGLISH;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public abstract class BaseTrinoCatalogTest
 {
     private static final Logger LOG = Logger.get(BaseTrinoCatalogTest.class);
+    protected static final ConnectorSession SESSION_WITH_PROPERTIES = TestingConnectorSession.builder()
```

Comment:
SESSION_WITH_PROPERTIES -> SESSION

Reply:
by doing this you avoid polluting the code with unnecessary changes which obfuscate anyway your changes.

Reply:
Oh, this will actually reduce the diff https://github.com/trinodb/trino/pull/24851#discussion_r1971504709